src/com/esotericsoftware/kryo/serializers/TaggedFieldSerializer.java

```diff
@@ -46,9 +49,12 @@
 	private int[] tags;
 	private int writeFieldCount;
 	private boolean[] deprecated;
+	/** If true, ignore unknown field tags. Otherwise throw KryoException. */
+	private boolean ignoreUnknownTags;

```

Comment:
Because config in `FieldSerializer` is constructed by `this.config = kryo.getFieldSerializerConfig().clone();`, so shall we also change the config in `Kryo` line 152 `private FieldSerializerConfig fieldSerializerConfig = new FieldSerializerConfig();` to `TaggedFieldSerializerConfig`?